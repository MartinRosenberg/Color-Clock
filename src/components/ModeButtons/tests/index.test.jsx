import { shallow } from 'enzyme'
import React from 'react'

import { ModeButtons } from '../index'

let handleSetMode

beforeEach(() => {
  handleSetMode = jest.fn()
})

describe('ModeButtons component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<ModeButtons handleSetMode={handleSetMode}/>)
    expect(wrapper).toMatchSnapshot()
  })
})


describe(`button`, () => {
  test('should call handleSetMode with correct mode', () => {})
})
