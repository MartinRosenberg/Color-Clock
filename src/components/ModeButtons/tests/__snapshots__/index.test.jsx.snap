// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModeButtons component should render correctly 1`] = `
<div>
  <button
    key="simple"
    onClick={[Function]}
    style={
      Object {
        "background": "#fff9",
        "border": 0,
        "borderRadius": "0.25rem",
        "margin": "0.5rem",
        "padding": "1rem",
      }
    }
  >
    Simple
  </button>
  <button
    key="rent"
    onClick={[Function]}
    style={
      Object {
        "background": "#fff9",
        "border": 0,
        "borderRadius": "0.25rem",
        "margin": "0.5rem",
        "padding": "1rem",
      }
    }
  >
    RENT
  </button>
  <button
    key="sunrise"
    onClick={[Function]}
    style={
      Object {
        "background": "#fff9",
        "border": 0,
        "borderRadius": "0.25rem",
        "margin": "0.5rem",
        "padding": "1rem",
      }
    }
  >
    Sunrise
  </button>
  <button
    key="unix"
    onClick={[Function]}
    style={
      Object {
        "background": "#fff9",
        "border": 0,
        "borderRadius": "0.25rem",
        "margin": "0.5rem",
        "padding": "1rem",
      }
    }
  >
    Unix
  </button>
</div>
`;
